tfgrid3/umbrel/zinit/config.yaml

```diff
@@ -0,0 +1,5 @@
+# configration for umbrel 
+exec: /bin/bash -xc '/scripts/umbrel-install.sh;' 
+oneshot: true
```

Comment:
as I understand, `oneshot` in `zinit` means if the service fails it won't be up again, for any reason it can be down, in that case, any other services depending on this service will block forever, no?

Reply:
any suggestion how to avoid that ?

Reply:
this may be helpful [comment](https://github.com/threefoldtech/tf-images/pull/125#discussion_r1104630161)